api/v2/views/base.py

```diff
@@ -24,8 +24,9 @@ def unresolved_requests_only(fn):
     @wraps(fn)
     def wrapper(self, request, *args, **kwargs):
         instance = self.get_object()
+        user_can_act = request.user.is_staff or request.user.is_superuser or CloudAdministrator.objects.filter(user=request.user.id)
```

Comment:
This queryset should includ `.count()` to ensure you get the truthy/falsy-ness you are looking for, and the Speed of an SQL lookup (Without having to generate an entire set of CA objects that you won't use!)

I will fix this one before merging in..